### PR TITLE
[R4ML-230] use public SparkR java methods

### PR DESCRIPTION
**Pull request by hungple**
Developer's Certificate of Origin 1.1

       By making a contribution to this project, I certify that:

       (a) The contribution was created in whole or in part by me and I
           have the right to submit it under the Apache License 2.0; or

       (b) The contribution is based upon previous work that, to the best
           of my knowledge, is covered under an appropriate open source
           license and I have the right under that license to submit that
           work with modifications, whether created in whole or in part
           by me, under the same open source license (unless I am
           permitted to submit under a different license), as indicated
           in the file; or

       (c) The contribution was provided directly to me by some other
           person who certified (a), (b) or (c) and I have not modified
           it.

       (d) I understand and agree that this project and the contribution
           are public and that a record of the contribution (including all
           personal information I submit with it, including my sign-off) is
           maintained indefinitely and may be redistributed consistent with
           this project or the open source license(s) involved.

### CODE REVIEW

File: R4ML/R/java.bridge.R [hunk 1]
```diff
@@ -38,10 +38,10 @@ log4j.Logger <- setRefClass("log4j.Logger",
     initialize = function(name = "org") {
       env <<- new.env()
       createLogger <- function(level) {
-        level_jref <- SparkR:::callJStatic("org.apache.log4j.Level", "toLevel", level)
+        level_jref <- sparkR.callJStatic("org.apache.log4j.Level", "toLevel", level)
```
Comment (bdwyer2) [inline R4ML/R/java.bridge.R]: We have decided to use `SparkR::` for all our SparkR functions. Can you please change this to `SparkR::sparkR.callJStatic`?

**Review by bdwyer2 — CHANGES_REQUESTED** [review]

File: R4ML/R/java.bridge.R (hunk 1 above)
Comment (hungple) [reply R4ML/R/java.bridge.R]: only for callJStatic?  How about callJMethod?

Comment (bdwyer2) [reply R4ML/R/java.bridge.R]: For both. Every public function from SparkR should start with `SparkR::` and every private function should start with `SparkR:::`.

**Comment by hungple** [comment]
@bdwyer2 I have added "::" for any place that we call "callJMethod" and "callJStatic".  Please review and approve. Thanks.

**Review by bdwyer2 — APPROVED** [review]

**Comment by CarmenRuppach** [comment]
@aloknsingh can this PR get merged?

**Pull request closed without merging**